distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2251,47 +1986,22 @@ public void testSingleUserDownloadSameProductSyncAndAsync() throws Exception {
                 RESOURCE_DOWNLOAD_ENDPOINT_ROOT.getUrl() + "?source=" + CSW_STUB_SOURCE_ID
                         + "&metacard=" + metacardId;
 
-        // Download product via async and then sync, should only call the stub server to download once
```

Comment:
Once we have the cancel download taken care of in the `tearDown()` method, we need to add the code that was removed from these tests back in so that these tests make sense again. Right now, they're basically all testing the same thing and not the method says they're testing.